frontend/components/KolideAce/KolideAce.jsx

```diff
@@ -90,7 +90,10 @@ class KolideAce extends Component {
           maxLines={20}
           name={name}
           onChange={onChange}
-          onLoad={onLoad}
+          onLoad={(editor) => {
+            editor.commands.removeCommands(['gotoline', 'find']);
+            onLoad(editor);
```

Comment:
Ah, this broke in the tests where `onLoad` was not set. Maybe this fixes it?
```suggestion
            onLoad && onLoad(editor);
```